jira-feature-collector/src/main/java/com/capitalone/dashboard/datafactory/jira/JiraDataFactoryImpl.java

```diff
@@ -254,8 +255,20 @@ public String setQuery(String query) {
 				}
 			} catch (Exception e) {
 				issues = new ArrayList<BasicProject>();
-				LOGGER.warn("No result was available from Jira unexpectedly - defaulting to blank response. The reason for this fault is the following:"
-						+ e.getCause());
+				if (e.getClass() == RestClientException.class) {
```

Comment:
This would definitely work; however, I believe that Hygieia now comes with its own Exception class that we can implement here.

That being said, if there is currently no other implementation of the Hygieia Exception class within this class, don't bother editing it; I will update the classes properly with my next PR.